.travis.yml

```diff
@@ -28,3 +28,7 @@ deploy:
 
     on:
         branch: master
+
+if: |
+    (branch = master OR type = pull_request) AND \
```

Comment:
Wouldn't build tags, but probably not a problem.

Reply:
This repository should not have tags, as it can't be independently released without the underlying environments it points to